spanner/google/cloud/spanner/snapshot.py

```diff
@@ -168,11 +174,17 @@ def __init__(self, session, read_timestamp=None, min_read_timestamp=None,
         if len(flagged) > 1:
             raise ValueError("Supply zero or one options.")
 
+        if multi_use and (min_read_timestamp or max_staleness):
```

Comment:
We just checked above that line that exactly one of the other options is not None.